### DISCUSSION TITLE
Qubev Smart charger - says its OCPP 1.6J compliant but I can't get it to talk

### DISCUSSION THREAD

**Discussion by MHKnowles** [lbbrhzn/ocpp#888] [Q&A]
Hi all,

My Rolec wallpoint decided to melt itself last week so I installed my newly bought QUBEV Smart charger today.  I managed to get the app working (after a while of working out that the charger doesn't support 5GHz wifi) but there's scant documentation for it and whilst it mentions how to configure the OCPP settings in the app i'm not confident i've configured it correctly (I'm an IT support guy for the last 35 years so i'm not a newb).

Every device sensor reads "unavailable" except Maximum Current which worked out it should be 32Amps but i'm not sure whether that's because the QUBEV app told it or because OCPP is slightly working.

So, two questions:
1) Has anybody else got one of these QUBEV Smart chargers working yet?
2) Where do I interrogate the logs for OCPP in HA?

Thanks,
Mike.

**Comment by jamiewhitham** [comment]
Hi Mike

Did you ever get it working?

I've tried setting OCPP server with ws://homeassistantipaddress:9000, with and without trailing slash, also first tried wss but then disabled the secure option in OCPP HACS plugin no joy. Reading the Qubev manual it says the ethernet cable is for OCPP only but seems to say WiFi can do both but not entirely sure of that. I am trying to do it via WiFi.

I'm only really after energy delivered in kWh.

Jamie

**Comment by MHKnowles** [comment]
Hi Jamie,

No sorry, I replaced the charger with a Zappi instead.

On 14 Dec 2025, at 11:54, jamiewhitham ***@***.***> wrote:

﻿

Hi Mike

Did you ever get it working?

I've tried setting OCPP server with ws://homeassistantipaddress:9000, with and without trailing slash, also first tried wss but then disabled the secure option in OCPP HACS plugin no joy. Reading the Qubev manual it says the ethernet cable is for OCPP only but seems to say WiFi can do both but not entirely sure of that. I am trying to do it via WiFi.

I'm only really after energy delivered in kWh.

Jamie
